dev-tracing/lttng_examples/dl/run.sh

```diff
@@ -0,0 +1,8 @@
+gcc -I. -fpic -c tpp.c
+gcc -shared -o libtpp.so tpp.o -llttng-ust -ldl
+
+gcc -c main.c
+gcc -o main main.o -ldl
+
+# LD_PRELOAD=./libtpp.so ./main
+./main
```

Comment:
Great! I see this starts and generates events on my system with lttng and starts in docker container that doesn't have lttng. That what we need.

Reply:
I would suggest add the dlopen to src/hotspot/os/linux/os_linux.cpp, `os::init_2` or nearby